ip_validation.py

```diff
@@ -0,0 +1,18 @@
+import socket
+import re
+
+input_ip = input("Insert your ip: ")

```

Comment:
Could you please verify your script as suggested in the `Check yourself` section in the task?